metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -1178,6 +1184,6 @@ object MetalsEnrichments
    * anything that is ESC (U+001B) plus [
    */
   def filterANSIColorCodes(str: String): String =
-    str.replaceAll("\u001B\\[[;\\d]*m", "")
+    str.replaceAll("\u001b\\[1A\u001b\\[K|\u001B\\[[;\\d]*m", "")
```

Comment:
I wonder if bazel it uses [cursor controls](https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797#cursor-controls) maybe we should try to cover all of them?

Reply:
Sounds like a good idea, but maybe we can do it in a follow up?